### PR TITLE
tools/etcd-tester: add kill majority test

### PR DESCRIPTION
**Pull request by yichengq**

### CODE REVIEW

File: tools/functional-tester/etcd-tester/failure.go [hunk 1]
```diff
@@ -16,10 +16,10 @@ package main
 
 type failure interface {
 	// inject the failure into the testing cluster
-	Inject(c *cluster) error
+	Inject(c *cluster, round int) error
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/failure.go]: document what is round

File: tools/functional-tester/etcd-tester/failure_killmaj.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			continue
+		}
+		if _, err := a.Restart(); err != nil {
+			return err
+		}
+	}
+	return c.WaitHealth()
+}
+
+func (f *failureKillMajority) killMap(size int, seed int) map[int]bool {
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/failure_killmaj.go]: why this is a method on failureKillMajority?

**Comment by yichengq** [comment]
PTAL

**Comment by yichengq** [comment]
ready for the review

**Comment by xiang90** [comment]
LGTM.

**Pull request merged**